branchout-yarn

```diff
@@ -5,9 +5,14 @@ set -e
 
 function runYarn() {
     if test -f "${PROJECTION_DIRECTORY}/.branchout/cacerts"; then
-      export NODE_EXTRA_CA_CERTS="${PROJECTION_DIRECTORY}/.branchout/cacerts"
+      if [ "$(command -v cygpath)" ]; then
+        NODE_EXTRA_CA_CERTS="$(cygpath --windows "${PROJECTION_DIRECTORY}/.branchout/cacerts")"
+      else
+        NODE_EXTRA_CA_CERTS="${PROJECTION_DIRECTORY}/.branchout/cacerts"
+      fi
 
       echo "NODE_EXTRA_CA_CERTS: ${NODE_EXTRA_CA_CERTS}"
+      export NODE_EXTRA_CA_CERTS=
```

Comment:
Should this be down a line, after the fi or in an else block?

Reply:
Thanks

Reply:
good spotting